pkgs/build-support/setup-hooks/move-docs.sh

```diff
@@ -6,7 +6,7 @@ preFixupHooks+=(_moveToShare)
 
 _moveToShare() {
     forceShare=${forceShare:=man doc info}
-    if [ -z "$forceShare" -o -z "$out" ]; then return; fi
+    if [[ -z "$forceShare" || -z "$out" ]]; then return; fi
```

Comment:
Wait, what exactly the first line here is supposed to do? It seems to check if `forceShare` is empty, and if so assign `man doc info` to it. Twice (once via default value assignment `:=`, second via the actual assignment). In any case, can `forceShare` ever be empty afterwards?

Reply:
You're right here, I've tested everything I could think of, and there is no way (that I could think of) that forceShare could be empty. Let me simplify that condition (I'll do it in the original PR).